hastexo/jobs.py

```diff
@@ -66,15 +66,33 @@ def run(self):
 
         # Get stacks to suspend
         with transaction.atomic():
-            stacks = Stack.objects.select_for_update().filter(
+            stacks_without_suspend_by = Stack.objects.select_for_update().filter(
```

Comment:
Whoops, now `flake8` chokes on this line. :) Might want to add `# noqa: E501` at the end rather than break this line.